runtime/onert/backend/train/KernelGenerator.cc

```diff
@@ -347,6 +348,32 @@ void KernelGenerator::visit(const ir::train::operation::Loss &node)
   }
 }
 
+void KernelGenerator::visit(const ir::train::operation::Pad &node)
+{
+  const auto input_index{node.getInputs().at(ir::operation::Pad::Input::INPUT)};
+  const auto pad_index{node.getInputs().at(ir::operation::Pad::Input::PAD)};
+  const auto output_index{node.getOutputs().at(0)};
+
+  auto input = _tensor_reg->getPortableTensor(input_index);
+  auto pad = _tensor_reg->getPortableTensor(pad_index);
+  auto output = _tensor_reg->getPortableTensor(output_index);
+
+  auto fn = std::make_unique<ops::PadLayer>();
+
+  IPortableTensor *value = nullptr;
```

Comment:
(Optional)
I guess this variable can be `const IPortableTensor *`
```suggestion
  IPortableTensor *value = nullptr;
```